FMS/Helpers/FacilityDetailDtoScalar.cs

```diff
@@ -82,6 +98,46 @@ public FacilityDetailDtoScalar(FacilityDetailDto facility)
         [DisplayFormat(DataFormatString = "{0:F6}")]
         public decimal Longitude { get; }
 
+        // The following properties only apply to Release Notifications

```

Comment:
These Boolean columns have values in the exported Excel spreadsheet even for non-RN Facilities.

Reply:
This goes back to my question of whether or not NULL values will cause problems if loaded into hidden parts of the "Add" and "Edit" pages.